### PR TITLE
[CB-211] Release Cohort Builder OWA version 1.0.0

### PR DESCRIPTION
**Pull request by aenkya**
# JIRA TICKET NAME:
[CB-211: Release Cohort Builder OWA version 1.1.0](https://issues.openmrs.org/browse/CB-211)
## SUMMARY:

The current release of Cohort Builder is 1.0.0-beta.
The task is to release an updated version of the Cohort Builder OWA 1.0.0

### CODE REVIEW

File: webpack.config.sample.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const WebpackOnBuildPlugin = require('on-build-webpack');
 
 const nodeModulesDir = path.resolve(__dirname, '../node_modules');
 
-const THIS_APP_ID = 'cohortbuilder-1.0.0-beta';
+const THIS_APP_ID = 'cohortbuilder-1.1.0';
```
Comment (dkayiwa) [inline webpack.config.sample.js]: The version should be 1.0.0

Comment (aenkya) [reply webpack.config.sample.js]: Alright thanks

**Review by segunolalive — COMMENTED** [review]
LGTM

**Comment by justmesam** [comment]
Looks Good 👍

**Review by malmike — COMMENTED** [review]
Looks good to me

**Pull request merged**